speech/cloud-client/pom.xml

```diff
@@ -0,0 +1,59 @@
+<!--
```

Comment:
Huh.
...okay, yeah - it's a quirk of how we run travis builds. ie for PRs, we don't run the full suite of tests - we see what files have changed, then cd into that directory before running tests. So for the PR, it'll run the right tests, but when running the full suite, it won't know about these tests without being in the parent pom.